discord/ext/tasks/__init__.py

```diff
@@ -134,15 +140,49 @@ def __get__(self, obj, objtype):
         if obj is None:
             return self
 
-        copy = Loop(self.coro, seconds=self.seconds, hours=self.hours, minutes=self.minutes,
-                               count=self.count, reconnect=self.reconnect, loop=self.loop)
+        copy = Loop(self.coro, seconds=self._seconds, hours=self._hours, minutes=self._minutes,
+                               count=self._count, time=self._time, reconnect=self.reconnect, loop=self.loop)
         copy._injected = obj
         copy._before_loop = self._before_loop
         copy._after_loop = self._after_loop
         copy._error = self._error
         setattr(obj, self.coro.__name__, copy)
         return copy
 
+    @property
+    def seconds(self):
+        """Optional[:class:`float`]: Read-only value for the number of seconds
+        between each iteration. ``None`` if an explicit ``time`` value was passed instead.
+        .. versionadded:: 1.7
```

Comment:
```suggestion
        """Optional[:class:`float`]: Read-only value for the number of seconds
        between each iteration. ``None`` if an explicit ``time`` value was passed instead.

        .. versionadded:: 1.7
```

Same for the other ones.